package.json

```diff
@@ -52,5 +52,5 @@
     "@types/react": "18.3.1",
     "@types/react-dom": "18.3.0"
   },
-  "packageManager": "yarn@3.5.0"
+  "packageManager": "yarn@4.3.1"
```

Comment:
It seems that Yarn v3.4.0 is already released.

Reply:
3.4.0 is previous version

Reply:
Misspelling, `4.4.0` of course is out.